src/background/Wallet/Wallet.ts

```diff
@@ -1538,7 +1543,7 @@ export class Wallet {
     const network = await fetchNetworkByChainId({
       preferences: Model.getPreferences(this.record),
       chainId,
-      apiEnv: 'current',
```

Comment:
Why do we need this change?

Reply:
Case on uniswap: we can be connected to a mainnet network, and dapp gets balance through the wallet rpc methods. But if in the wallet we have testmode enabled, then here we won't be able to get data for the mainnet network to which the dapp is currently connected, and the dapp's UI will appear to be broken (unable to display balance)